### ISSUE TITLE
Breaks JS in Chrome when Hardware Acceleration is disabled

### ISSUE THREAD

**Issue by jsunsawyer** [martinlaxenaire/curtainsjs#80]
**Describe the bug**
When Hardware Acceleration is disabled, javascript breaks for the entire page.

This can be tested on https://www.curtainsjs.com/

**Screenshots**
![Screen Shot 2021-03-15 at 9 11 52 AM](https://user-images.githubusercontent.com/1007191/111184743-8178b380-856e-11eb-8420-30e85eb86855.png)

**Desktop (please complete the following information):**
 - OS: macOS 10.15.7
 - Browser: Chrome
 - Version: 89.0.4389.82

**Additional context**
Hardware Acceleration must be disabled.

**Comment by martinlaxenaire** [comment]
Hi @jsunsawyer, 

I've improved the WebGL context creation failure event handling, so the lib shouldn't break the whole page when it happens.

However your javascript still might break when you try to access things like plane uniforms if the context creation has failed for example.
To prevent this, I've added a `onSuccess` method to the `Curtains` class, inside which you might want to add all your plane instanciation logic. See documentation: https://www.curtainsjs.com/curtains-class.html#on-success

Cheers,
